Traceback (most recent call last):
  File "C:\Users\Sean\anaconda3\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\Users\Sean\anaconda3\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\Sean\anaconda3\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\Sean\anaconda3\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\Sean\anaconda3\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "C:\Users\Sean\anaconda3\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\Users\Sean\anaconda3\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\Sean\anaconda3\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import test
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mImportError[0m                               Traceback (most recent call last)
Input [1;32mIn [1][0m, in [0;36m<cell line: 1>[1;34m()[0m
[1;32m----> 1[0m [38;5;28;01mimport[39;00m [38;5;21;01mtest[39;00m

File [1;32m~\OneDrive - Virginia Tech\documents\teaching\Course ebooks\Python-for-MSE\lessons_3114\Week 2\test.py:3[0m, in [0;36m<module>[1;34m[0m
[0;32m      1[0m [38;5;28;01mimport[39;00m [38;5;21;01mnumpy[39;00m [38;5;28;01mas[39;00m [38;5;21;01mnp[39;00m
[0;32m      2[0m [38;5;28;01mimport[39;00m [38;5;21;01mpandas[39;00m [38;5;28;01mas[39;00m [38;5;21;01mpd[39;00m
[1;32m----> 3[0m [38;5;28;01mimport[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mpyplot[39;00m [38;5;28;01mas[39;00m [38;5;21;01mplt[39;00m
[0;32m      5[0m [38;5;28;01mdef[39;00m [38;5;21mln[39m(x):
[0;32m      6[0m     [38;5;28;01mreturn[39;00m(np[38;5;241m.[39mlog(x))

File [1;32m~\anaconda3\lib\site-packages\matplotlib\__init__.py:109[0m, in [0;36m<module>[1;34m[0m
[0;32m    105[0m [38;5;28;01mfrom[39;00m [38;5;21;01mpackaging[39;00m[38;5;21;01m.[39;00m[38;5;21;01mversion[39;00m [38;5;28;01mimport[39;00m parse [38;5;28;01mas[39;00m parse_version
[0;32m    107[0m [38;5;66;03m# cbook must import matplotlib only within function[39;00m
[0;32m    108[0m [38;5;66;03m# definitions, so it is safe to import from it here.[39;00m
[1;32m--> 109[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m [38;5;28;01mimport[39;00m _api, _version, cbook, docstring, rcsetup
[0;32m    110[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcbook[39;00m [38;5;28;01mimport[39;00m MatplotlibDeprecationWarning, sanitize_sequence
[0;32m    111[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcbook[39;00m [38;5;28;01mimport[39;00m mplDeprecation  [38;5;66;03m# deprecated[39;00m

File [1;32m~\anaconda3\lib\site-packages\matplotlib\rcsetup.py:27[0m, in [0;36m<module>[1;34m[0m
[0;32m     25[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmatplotlib[39;00m [38;5;28;01mimport[39;00m _api, cbook
[0;32m     26[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcbook[39;00m [38;5;28;01mimport[39;00m ls_mapper
[1;32m---> 27[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcolors[39;00m [38;5;28;01mimport[39;00m Colormap, is_color_like
[0;32m     28[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mfontconfig_pattern[39;00m [38;5;28;01mimport[39;00m parse_fontconfig_pattern
[0;32m     29[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01m_enums[39;00m [38;5;28;01mimport[39;00m JoinStyle, CapStyle

File [1;32m~\anaconda3\lib\site-packages\matplotlib\colors.py:51[0m, in [0;36m<module>[1;34m[0m
[0;32m     49[0m [38;5;28;01mfrom[39;00m [38;5;21;01mnumbers[39;00m [38;5;28;01mimport[39;00m Number
[0;32m     50[0m [38;5;28;01mimport[39;00m [38;5;21;01mre[39;00m
[1;32m---> 51[0m [38;5;28;01mfrom[39;00m [38;5;21;01mPIL[39;00m [38;5;28;01mimport[39;00m Image
[0;32m     52[0m [38;5;28;01mfrom[39;00m [38;5;21;01mPIL[39;00m[38;5;21;01m.[39;00m[38;5;21;01mPngImagePlugin[39;00m [38;5;28;01mimport[39;00m PngInfo
[0;32m     54[0m [38;5;28;01mimport[39;00m [38;5;21;01mmatplotlib[39;00m [38;5;28;01mas[39;00m [38;5;21;01mmpl[39;00m

File [1;32m~\anaconda3\lib\site-packages\PIL\Image.py:103[0m, in [0;36m<module>[1;34m[0m
[0;32m    100[0m     [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m [38;5;28;01mimport[39;00m _imaging [38;5;28;01mas[39;00m core
[0;32m    102[0m     [38;5;28;01mif[39;00m __version__ [38;5;241m!=[39m [38;5;28mgetattr[39m(core, [38;5;124m"[39m[38;5;124mPILLOW_VERSION[39m[38;5;124m"[39m, [38;5;28;01mNone[39;00m):
[1;32m--> 103[0m         [38;5;28;01mraise[39;00m [38;5;167;01mImportError[39;00m(
[0;32m    104[0m             [38;5;124m"[39m[38;5;124mThe _imaging extension was built for another version of Pillow or PIL:[39m[38;5;130;01m\n[39;00m[38;5;124m"[39m
[0;32m    105[0m             [38;5;124mf[39m[38;5;124m"[39m[38;5;124mCore version: [39m[38;5;132;01m{[39;00m[38;5;28mgetattr[39m(core, [38;5;124m'[39m[38;5;124mPILLOW_VERSION[39m[38;5;124m'[39m, [38;5;28;01mNone[39;00m)[38;5;132;01m}[39;00m[38;5;130;01m\n[39;00m[38;5;124m"[39m
[0;32m    106[0m             [38;5;124mf[39m[38;5;124m"[39m[38;5;124mPillow version: [39m[38;5;132;01m{[39;00m__version__[38;5;132;01m}[39;00m[38;5;124m"[39m
[0;32m    107[0m         )
[0;32m    109[0m [38;5;28;01mexcept[39;00m [38;5;167;01mImportError[39;00m [38;5;28;01mas[39;00m v:
[0;32m    110[0m     core [38;5;241m=[39m DeferredError([38;5;167;01mImportError[39;00m([38;5;124m"[39m[38;5;124mThe _imaging C module is not installed.[39m[38;5;124m"[39m))

[1;31mImportError[0m: The _imaging extension was built for another version of Pillow or PIL:
Core version: "9.2.0"
Pillow version: 9.2.0
ImportError: The _imaging extension was built for another version of Pillow or PIL:
Core version: "9.2.0"
Pillow version: 9.2.0

